packages/integration/src/gitlab/config.ts

```diff
@@ -54,6 +54,14 @@ export type GitLabIntegrationConfig = {
    * If no baseUrl is provided, it will default to `https://${host}`
    */
   baseUrl: string;
+
+  /**
+   * The relative path for gitlab self hosted installations
+   * described here https://docs.gitlab.com/ee/install/relative_url.html
+   *
+   * If no baseUrl is provided, it will default to `/`
+   */
+  relativePath?: string;
```

Comment:
- My thought here was to calculate relativePath once and use it wherever it's needed as an integration property, instead of having to get it multiple times, but also happy to reconsider having a function
- I agree, slash was probably there for convenience when building URLs, but can probably get rid of it.